treeherder/model/management/commands/cache_failure_history.py

```diff
@@ -40,23 +44,15 @@ def handle(self, *args, **options):
         for day in range(days):
             push_date = datetime.datetime.now().date() - datetime.timedelta(days=day)
 
-            int_hist, cache_key = get_history(
-                4,
-                push_date,
-                intermittent_history_days,
-                option_map,
-                repository_ids,
-                True)
-            self.debug('Cached failure history for {}'.format(cache_key))
-
-            fbc_hist, cache_key = get_history(
-                2,
-                push_date,
-                fixed_by_commit_history_days,
-                option_map,
-                repository_ids,
-                True)
-            self.debug('Cached failure history for {}'.format(cache_key))
+            get_history(4, push_date, intermittent_history_days, option_map, repository_ids, True)
```

Comment:
This was the bug.  I had originally returned the `cache_key` as a tuple.  But forgot to update this code when I stopped returning it.